odata-url-conventions/5 Query Options.md

```diff
@@ -1706,7 +1706,7 @@ $filter=[FirstName,LastName] in [["John","Doe"],["Jane","Smith"]]
 ```
 :::
 
-Entities are represented as structured literals as described in [OData-JSON #odata-json-format#Entity].
+Entities are represented as structured literals as described in [OData-JSON, section #odata-json-format#Entity].
```

Comment:
Why do we have to write "section" everywhere?

Reply:
We write "section" for internal references as well. [OData-JSON, 6] could also mean a page. But let the TC decide.

Reply:
I thought `[subsection ##SubsectionHeading]` and `[example ##aggregation]` were hints for the reference resolver to distinguish between section references and example references, not actual output text.

If I write `[OData-CSDL #odata-json-format#Entity]`, would the text say "OData-CSDL" and the link point to OData-JSON?

Being lazy I'd prefer to write `[#odata-json-format#Entity]` and get both the text "OData JSON, section 6" and the correct link. I'd even be willing to write more hashes if I don't have to repeat the spec name, for example `[###odata-json-format#Entity]`.

Reply:
> If I write `[OData-CSDL #odata-json-format#Entity]`, would the text say "OData-CSDL" and the link point to OData-JSON?

I use this "redundancy" to write `[OData-CSDL, #odata-csdl-json#Annotations]`, because the link goes to the JSON variant, not the XML variant.

Reply:
I want to see it in the output text, I'd prefer not having to write it _unless_ we want to allow referencing _examples_ in other documents in the future.

Reply:
Long syntax `[OData-Protocol, section #OData-Protocol#SectionHeading]`. (Can also be used for examples.)
Short variant for section references: `[#OData-Protocol#SectionHeading]`